### ISSUE TITLE
debops.locales missing on Ansible Galaxy?

### ISSUE THREAD

**Issue by Skeen** [debops/debops#244]
TL:DR; `debops.locales` is not found on [Ansible Galaxy](https://galaxy.ansible.com/debops/?role=locales)

I'm trying to execute the example playbook from `debops.postgresql_server` (see [here](https://docs.debops.org/en/latest/ansible/roles/debops.postgresql_server/getting-started.html#example-playbook)).

I'm executing the playbook and thus roles as standalone; as I do this, I get errors regarding the missing roles in the example playbook. I can solve these using an `ansible-galaxy install debops.xxxx` command.

I cannot however do this with the debops.locales role, as it does not seem to appear on Ansible Galaxy.
Is this by design, or a mistake?

**Comment by drybjed** [comment]
The initial plan with DebOps monorepo was that for convenience, roles would be exported to their own repositories on GitHub and published on Ansible Galaxy that way. This would require writing a few scripts that would extract the roles, create GitHub repositories, register them on Galaxy, etc. That was supposed to happen after the move was completed when we would know what we have to work with

Some time after the move to the monorepo was complete I learned that Ansible Galaxy team [s working on support for multiple roles in one repository](https://github.com/ansible/galaxy/issues/244). This was good news since with this feature implemented all of the overhead with custom export scripts would disappear. Therefore I decided to wait for this feature to be complete with exporting the roles. I suspect that there will need to be some reorganization of the DebOps monorepo to integrate it better with new Galaxy features, but that's relatively easy to deal with.

For now you could clone the DebOps monorepo to somewhere in your filesystem and add path to `ansible/roles/` directory to the `roles_path` option in `ansible.cfg` - this should allow Ansible to use DebOps roles as needed.

**Comment by lingfish** [comment]
Hi, it would be good if doco was updated here, until Galaxy's changes are released.  I'm properly confused as to how to standalone install DebOps stuff now, due to missing/deeply nested `env/` directories etc.

**Comment by drybjed** [comment]
The [installation guide](https://docs.debops.org/en/master/user-guide/install.html) has been updated with explanation on how to install DebOps roles via Ansible Galaxy. Closing as resolved.
